views/beers.hbs

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/stylesheets/styles.css">
+    <title>Beers</title>
+</head>
+<body>
+<div>
+  {{#each beersFromApi}}
+    <h3>{{this.name}}</h3>
+    <p>{{this.tagline}}</p>
+    <p>{{this.description}}</p>
+    <img src='{{this.image_url}}' alt='' />
```

Comment:
Here, since the response from the API is retrieving a string with a link to the image, you should not pass the `this.image_url` inside double quotes.